R/processing.R

```diff
@@ -926,25 +962,30 @@ run_qgis <- function(alg = NULL, params = NULL, check_params = TRUE,
     message(msg)
   }
   
-  
   # load output
   if (!is.null(load_output)) {
     ls_1 <- lapply(load_output, function(x) {
       
       fname <- ifelse(dirname(x) == ".", 
                       file.path(tmp_dir, x),
                       x)
-       if (!file.exists(fname)) {
+      if (!file.exists(fname)) {
         stop("Unfortunately, QGIS did not produce: ", x)
       }
-     
-      test <- try(expr = 
-                    rgdal::readOGR(dsn = dirname(fname),
-                                   layer = gsub("\\..*", "", 
-                                                basename(fname)),
-                                   verbose = FALSE),
-                  silent = TRUE
-      )
+      
+      if (input.sf == FALSE) {
+        test <- try(expr = rgdal::readOGR(dsn = dirname(fname),
```

Comment:
remove package tag `::` -> not used in R packages.  
Specify single functions in the documentation using `@importFrom rgdal readOGR`